src/components/directory-content.js

```diff
@@ -882,13 +937,21 @@ const DirectoryContent = () => {
                 open={openDeleteElementDialog}
                 onClose={handleCloseDeleteElement}
                 onClick={handleClickDeleteElement}
-                title={useIntl().formatMessage(
-                    { id: 'deleteElement' },
-                    { stn: getSelectedChildren().length }
-                )}
-                message={useIntl().formatMessage({
-                    id: 'genericConfirmQuestion',
-                })}
+                title={
```

Comment:
must be managed inside the deleteDialog component not in directory-content

Reply:
done